cloudflared/rootfs/etc/services.d/cloudflared/run

```diff
@@ -15,8 +15,13 @@ if bashio::config.true 'quick_tunnel'; then
     bashio::log.info "Connecting Cloudflared Quick Tunnel..."
     exec cloudflared --no-autoupdate \
     tunnel --loglevel="${CLOUDFLARED_LOG}" --url "${ha_service_protocol}://homeassistant:$(bashio::core.port)"
-else
 
+elif bashio::config.has_value 'tunnel_token' ; then
+    tunnel_token=$(bashio::config 'tunnel_token')
+    exec cloudflared --no-autoupdate \
```

Comment:
I assume that the tunnel option also means ignoring any config files that might exist. Cannot verify at the moment though, but let's go with it